src/AV/Video.cpp

```diff
@@ -1367,10 +1368,10 @@ int Video::addTimer(int delay)
 
 bool Video::addPacket(PacketQueue *queue, AVPacket *packet)
 {
-    av_dup_packet(packet);
+    AVPacket *dup = av_packet_clone(packet);
```

Comment:
According to the doxy guide: 'av_packet_clone': `is a shortcut for 'av_packet_alloc()' + 'av_packet_ref()'; http://www.ffmpeg.org/doxygen/3.0/group__lavc__packet.html#gab367656bda4319f1b2358092e2d5e624

Please advice.